src/app/profile/profile.component.html

```diff
@@ -3,78 +3,34 @@ <h1 class="text-center text-info font-weight-bold">
-          <div class="row justify-content-center">
-            <div class="col-8">
-              <a href="calendar" alt="Events Calendar" title="Events Calendar"
-                ><img
-                  src="assets/volunteer-banner.svg"
-                  alt="Welcome To The Volunteer Calendar"
-                  class="rounded img-fluid text-centered mx-auto"
-              /></a>
-            </div>
-            <div class="col px-3 mx-auto d-none d-xl-block">
-              <h3 class="font-weight-bold text-light">
-                Welcome to the Volunteer Calendar!
-              </h3>
-              <p class="text-light">
-                Please explore and find organizations or upcoming events to
-                join.
-              </p>
-            </div>
-          </div>
-        </ng-template>
-        <ng-template ngbSlide>
-          <div class="row justify-content-center">
-            <div class="col-8">
-              <a href="calendar" alt="Events Calendar" title="Events Calendar"
-                ><img
-                  src="assets/volunteer-banner.svg"
-                  alt="Welcome To The Volunteer Calendar"
-                  class="rounded img-fluid text-centered mx-auto"
-              /></a>
-            </div>
-            <div class="col px-3 mx-auto d-none d-xl-block">
-              <h3 class="font-weight-bold text-light">
-                Welcome to the Volunteer Calendar!
-              </h3>
-              <p class="text-light">
-                Please explore and find organizations or upcoming events to
-                join.
-              </p>
-            </div>
-          </div>
-        </ng-template>
-        <ng-template ngbSlide>
-          <div class="row justify-content-center">
-            <div class="col-8">
-              <a href="calendar" alt="Events Calendar" title="Events Calendar"
-                ><img
-                  src="assets/volunteer-banner.svg"
-                  alt="Welcome To The Volunteer Calendar"
-                  class="rounded img-fluid text-centered mx-auto"
-              /></a>
-            </div>
-            <div class="col px-3 mx-auto d-none d-xl-block">
-              <h3 class="font-weight-bold text-light">
-                Welcome to the Volunteer Calendar!
-              </h3>
-              <p class="text-light">
-                Please explore and find organizations or upcoming events to
-                join.
-              </p>
-            </div>
-          </div>
-        </ng-template>
-      </ngb-carousel>
+  <div class="user">
+    <div>
+      <img
+        src="https://is5-ssl.mzstatic.com/image/thumb/Purple118/v4/ea/69/7c/ea697c12-6de6-22bd-244f-cc48e1a49318/mzl.iayvqrek.png/246x0w.jpg"
+        alt="batman-portrait">
+    </div>
+    <div>
+      <strong>About Me:</strong> <br>
+      <div class="blurb">
+        <span>My volunteering passion is dressing up as Batman to cheer up kids stuck in prisons/ I mean schools. I have
```

Comment:
formatting seems a little odd here, probably fine for now; but if this is for page aesthetics, consider a more dynamic solution in the future to account for a range of devices/browsers/etc (maybe it just looks funky on my machine though /shrug)